lib/plugins/aws/provider.js

```diff
@@ -730,6 +738,7 @@ class AwsProvider {
               anyOf: ['REGIONAL', 'EDGE', 'PRIVATE'].map(caseInsensitive),
             },
             environment: { $ref: '#/definitions/awsLambdaEnvironment' },
+            eventBridge: eventBridgeConfig,
```

Comment:
Let's configure this schema inline (in case of schema it's valuable for fast inspection to have everything inline - taking out into vars, makes sense if there's some re-use case)